lms/templates/fields/field_dropdown.underscore

```diff
@@ -4,6 +4,12 @@
     </label>
 <% } %>
 
+<% if (screenReaderTitle) { %>
+    <label class="sr" for="u-field-select-<%- id %>">
+        <%- screenReaderTitle %>
+    </label>
+<% } %>
```

Comment:
@symbolist I guess the second approach you suggested here would be a better one as I think it might not be good to have two separate elements for one field that could be used many times in future as you said. So I going with second one and try to implement it.